src/pages/AddProposal/AddProposal.container.tsx

```diff
@@ -88,20 +78,24 @@ export const AddProposalContainer = ({
     };
   }, [dispatch, govSelector]);
 
+  const onGovernorChange = (gov: string) => {
```

Comment:
I would wrap the function inside the `useCallback` hook since it's later used in the dependency list of `useEffect`